Android/app/src/main/java/com/cs4125/bikerentalapp/model/entity/BikeInfo.java

```diff
@@ -0,0 +1,17 @@
+package com.cs4125.bikerentalapp.model.entity;
+
+public class BikeInfo extends  AbstractBikeInfo {
+    private String type;
+    private String l;
```

Comment:
If you could make this variable name more descriptive it would be good. Took me a while to figure out what its purpose was. Something like locationName would probably work.